akka-diagnostics/src/main/scala/akka/dispatch/affinity/PoolTypeAccessor.scala

```diff
@@ -1,4 +1,6 @@
-package akka.dispatch.affinity
+package akka
+
+import akka.dispatch.affinity.AffinityPool
 
 object PoolTypeAccessor {
```

Comment:
Is this needed? I think it was a trick to access `private[akka]` class but now we have the code in `akka` package so shouldn't be needed?

Reply:
removed